app/src/main/res/values/dimens.xml

```diff
@@ -151,4 +151,6 @@
     <dimen name="popup_arrow_size">16dp</dimen>
 
     <dimen name="language_radius">4dp</dimen>
+    <dimen name="tag_radius">20dp</dimen>
```

Comment:
Why tag items radius differs from language items radius?

Reply:
It is design decision http://prntscr.com/hgfhd3